vignettes/col_plan.Rmd

```diff
@@ -14,6 +14,283 @@ knitr::opts_chunk$set(
+
+Let's consider an example dataset with multiple `column` variables:
+
+```{r}
+dat <- tribble(
+    ~group,     ~label,        ~span2,  ~span1,     ~my_col,    ~parm,   ~val,
+    "g1", "rowlabel1",  "column cols", "cols 1,2", "col1"  ,  "value",    1,
+    "g1", "rowlabel1",  "column cols", "cols 1,2", "col2"  ,  "value",    1,
+    "g1", "rowlabel1",             NA,         NA, "mycol3",  "value",    1,
+    "g1", "rowlabel1",  "column cols", "col 4"   , "col4"  ,  "value",    1,
+    "g1", "rowlabel1",             NA,         NA, "mycol5",  "value",    1,
+    "g1", "rowlabel2",  "column cols", "cols 1,2", "col1"  ,  "value",    2,
+    "g1", "rowlabel2",  "column cols", "cols 1,2", "col2"  ,  "value",    2,
+    "g1", "rowlabel2",             NA,        NA , "mycol3",  "value",    2,
+    "g1", "rowlabel2",  "column cols", "col 4"   , "col4"  ,  "value",    2,
+    "g1", "rowlabel2",             NA,         NA, "mycol5",  "value",    2,
+    "g2", "rowlabel3",  "column cols", "cols 1,2", "col1"  ,  "value",    3,
+    "g2", "rowlabel3",  "column cols", "cols 1,2", "col2"  ,  "value",    3,
+    "g2", "rowlabel3",             NA,         NA, "mycol3",  "value",    3,
+    "g2", "rowlabel3",  "column cols", "col 4"   , "col4"  ,  "value",    3,
+    "g2", "rowlabel3",             NA,         NA, "mycol5",  "value",    3,
+  )
+```
+
+Notice that our `col_plan` is considerably simpler and our higher level columns are repurposed as column spanners:
+
+```{r}
+tfrmt(
+    group = group,
+    label = label,
+    param = parm,
+    values = val,
+    column = c(span2, span1, my_col),
+    body_plan = body_plan(
+      frmt_structure(group_val = ".default", label_val = ".default", frmt("x"))
+    ),
+    col_plan = col_plan(
+      group, 
+      label, 
+      starts_with("col"),
+      new_col_3 = mycol3,  
+      -mycol5            
+    )
+  ) %>% 
+  print_to_gt(dat)
+```
+
+
+When there are multiple `column` variables provided, `tlang` concatenates them behind the scenes using a special delimiter. Therefore, the underlying column names in the final table are a combination of the spanner(s) and the lowest level column label. In order to change the order of spanning groups, we may use`contains()` to reference any portion of those names. 
```

Comment:
Not sure if we want to share the details on how `tlang` behaves, but understand why to make it so folks understand how they can use `contains()`. 

```suggestion
When there are multiple `column` variables provided, `tlang` concatenates them behind the scenes using a special delimiter. Therefore, the underlying column names in the final table are a combination of the spanner(s) and the lowest level column label. In order to change the order of spanning groups, we may use `contains()` to reference any portion of those names. 
```

Reply:
I'm going to revise so it's somewhere in the middle.